src/main/scala/elo/Bot.scala

```diff
@@ -116,7 +116,7 @@ object PingPongBot extends App {
           challenger <- challengerOpt
           challengee <- challengeeOpt
         } yield {
-          reportLoss(challenger, challengee)
+          if(challenger == challengee) "You play yourself, dummy!" else reportLoss(challenger, challengee)
```

Comment:
I'd personally move this into the reportLoss method, but otherwise nice job with the quick bug fix turnaround lol

Reply:
Ok I'll do that